src/templates/tag.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { graphql, Link } from 'gatsby'
+import styled from 'styled-components'
+
+import Layout from '../components/Layout'
+import SEO from '../components/SEO'
+import { IMember, Subset } from '../shared/Icons/types'
+import {
+  Section,
+  H1,
+  P,
+  Tags,
+  GitHubIcon,
+  LinkedInIcon,
+  LinkIcon,
+  Row,
+  Col,
+  BookOpenIcon,
+  HomeIcon,
+  CalendarIcon,
+  LogOutIcon,
+  WideContainer,
+  MediumContainer,
+} from '../shared'
+import {
+  M2,
+  BORDER_RADIUS_LG,
+  M1,
+  M3,
+  maxWidth,
+  PHONE,
+  M4,
+} from '../constants/measurements'
+import { DARK_GRAY } from '../constants/colors'
+
+const TagTemplate = ({ data, location, pageContext }) => {
+  const tag = data.ghostTag
+  const posts = data.allGhostPost.edges
+
+  return (
+    <Layout>
+      <SEO title={tag.name} />
```

Comment:
Would be good to destructure name and such above the render method then just use it directly (similar to comment made on other file above)